code/modules/research/xenobiology/crossbreeding/_status_effects.dm

```diff
@@ -57,6 +57,144 @@
 	owner.visible_message("<span class='warning'>[owner]'s gel coating liquefies and dissolves away.</span>",
 		"<span class='notice'>Your gel second-skin dissolves!</span>")
 
+/datum/status_effect/slimerecall
+	id = "slime_recall"
+	duration = -1 //Will be removed by the extract.
+	alert_type = null
+	listening = TRUE
+	var/interrupted = FALSE
+	var/mob/target
+	var/icon/bluespace
+
+/datum/status_effect/slimerecall/on_apply()
+	RegisterEffectSignal(COMSIG_LIVING_RESIST, .proc/resistField)
```

Comment:
I'm absolutely horrified this was let through earlier, half assing the conversion of status effects to components is terrible.

Reply:
...aw.
i put a lot of work into that.
any suggestions?

Reply:
It's not appropriate in this pr but the status effect type path should have been removed entirely

Reply:
With #38537 removing this, there are heavy conflicts between the two prs. I'll leave it up to @Cyberboss whether he wants to fix the conflicts or make @fludd12 handle it depending on merge order.

Reply:
I'll use cyberboss' method - no sense halting another PR because of me.